plugins/woocommerce/includes/wc-ipp-functions.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * WooCommerce In-Person Payments functions
+ *
+ * @package WooCommerce\Functions
+ * @version 7.0
+ */
+
+/**
+ * Returns if order is eligible to accept In-Person Payments
+ *
+ * @param WC_Order $order order that the conditions are checked for.
+ *
+ * @return true if order is eligible, false otherwise
```

Comment:
```suggestion
 * @return bool True if order is eligible, false otherwise.
```

The `@return` type should be shown as `bool`. Right now my IDE interprets this as the function always returning specifically `true`. Same goes for the other functions in this file.